content/applications/general/digest_emails.rst

```diff
@@ -0,0 +1,116 @@
+:show-content:
+
+=============
+Digest Emails
+=============
+
+**Digest Emails** are periodic snapshots sent to your organization via email that include high-level
+information about how your business is performing.
+
+Navigate to Digest Emails by going to :menuselection:`Settings --> General Settings --> Statistics`,
+then activate the **Digest Emails** feature and click on save.
+
+.. image:: digest_emails/digest-email-settings.png
+   :align: center
+   :alt: Digest Emails section inside General Settings.
+
+You can control a variety of settings for your Digest Emails, such as:
+
+- which KPIs are shared in the Digest
+- how often Digest Emails are sent
+- who in your organization receives Digest Emails
+- creating custom Digest Email templates
+- adding additional KPIs (Studio required)
+
+.. note::
+   By default, Digest Email is *enabled*, and *Your Odoo Periodic Digest* serves as the primary
+   template, which includes all KPI measurements across your Odoo database and is sent daily to
+   administrators.
+
+.. _digest-emails/customize-digest:
+
+Customize *Your Odoo Periodic Digest*
+=====================================
+
+To customize the default Digest Email (*Your Odoo Periodic Digest*), go to :menuselection:`Settings
+--> General Settings --> Statistics --> Digest Email`, select *Your Odoo Periodic Digest* and click
+on the *external link* next to the dropdown selection.
+
+A popup window appears and presents a variety of editable settings, which include:
+
+- **Digest Title** - what you want your Digest Email to be called
+- **Periodicity** - control the regimen in how often Digest Emails are sent
+- **KPIs** - check/uncheck each calculated KPI that appears in Digest Emails
+- **Recipients** - add/remove users who receive your Digest Emails
+- **Custom** - add your own KPIs (Studio required)
+
+.. image:: digest_emails/periodic-digest.png
+   :align: center
+   :alt: Customize default Digest Email settings and custom KPIs.
+
+.. _digest-emails/custom-emails:
+
+Custom digest emails
+====================
+
+To do so, click on **-> Configure Digest Emails** and then **Create**.
```

Comment:
I'm not sure if mentioning the arrow here is worth it. It might be removed/changed with another symbol in Odoo and require updating this page.
```suggestion
To do so, click on **Configure Digest Emails** and then **Create**.
```